src/components/useStore/useStore.js

```diff
@@ -27,6 +28,10 @@ const useStore = create(
 				};
 			});
 		},
+		bookmarkStatus: '',
+		updateBookmark: status => {
+			set({ bookmarkStatus: status });
+		},
```

Comment:
set bookmarkStatus initial to `null` and also use `null` when you want to reset the state like this: `updateBookmark(null)`. This resolves bookmarkStatus to be `truthy` or `falsy` which seems to fit better to the states purpose.

Reply:
I'm not sure I understand the last comment